core/src/parking_lot.rs

```diff
@@ -16,6 +16,39 @@ use core::{
 use smallvec::SmallVec;
 use std::time::{Duration, Instant};
 
+cfg_if! {
+    if #[cfg(target_arch = "wasm32")] {
+        use core::ops::Add;
+
+        #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Debug, Hash)]
+        struct DummyInstant(Duration);
+
+        impl DummyInstant {
+            pub fn now() -> DummyInstant {
+                DummyInstant::zero()
+            }
+
+            pub const fn zero() -> DummyInstant {
+                DummyInstant(Duration::from_secs(0))
+            }
```

Comment:
This method is only used in one place, which means that:
- It doesn't need to be `pub`.
- You may as well just inline it into the `add` function.

Reply:
Yep, so I could reuse the logic there.

Reply:
Removing `pub`. Good point!

Reply:
I ll wait for Travis to finish to see if all goes green.